corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -647,6 +681,11 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
         self.groupId = groupNum++;
         self.rel_ix = ko.observable(relativeIndex(self.ix()));
         self.isRepetition = parent.parent instanceof Repeat;
+        if (Object.hasOwn(self, 'delete')) {
+            self.showDelete = self.delete();
```

Comment:
Is there a `delete` attribute coming from formplayer that indicates whether or not a group can be deleted?

Reply:
Yes, but only with the new changes. I added a comment to simplify this once the formplayer deploy is done.